### ISSUE TITLE
phoneme_silence problem

### ISSUE THREAD

**Issue by hairygael** [artibex/piper-http#4]
Hello, when building my docker image, it doesn't seem to take in account the options for phoneme_silence which can be added to the onnx.json voices. If I run piper directly with python the result is working but not with the docker image.
[fr_FR-siwis-medium.onnx.json](https://github.com/user-attachments/files/17349537/fr_FR-siwis-medium.onnx.json)

**Comment by artibex** [comment]
Interesting...

Are you setting up the container, editing the file and restarting the container?
If so, the container will pull the files again if not existing when I correctly recall it.

**Comment by hairygael** [comment]
![Capture d’écran 2024-10-16 123826](https://github.com/user-attachments/assets/accb0e91-4775-4e7f-96c4-5c7d464d8323)
I run `docker build -t piper .` to create the image which has access to the link of the modified onnx.json
Then I use `docker run --name siwis -p 5000:5000 piper

I just checked in the container and the onnx.json does have the modifications.

**Comment by artibex** [comment]
Ok that's interesting...
Maybe the docker container is referencing a different .json file when running? Are there any other files that could be used other then your target files?

**Comment by hairygael** [comment]
I don't think this is the case, because I tried modifying "length_scale" and I can tell that is taken in account. (The voice is then very slowly speaking). I am using the docker image to be used by another software called Myrobotlab, I wonder if that could be the part that prevents the use of "phoneme_silence" and "sentence_silence" in my onnx.json .("sentence_silence" also doesn't seem to do effect.)

**Comment by artibex** [comment]
I'v created a client file for testing the server system: https://github.com/artibex/piper-http/blob/main/client/piper-tts.py 

You could make a test run with that so see if the output is different but I doubt that

**Comment by hairygael** [comment]
Thanks for your patience, I am not sure how to use the client file you've created...

**Comment by artibex** [comment]
No worries!
You need a python environment for that. You can use a IDE like VS Code for running it or use your console. Just put the file somewere to reference it.

It's importent to set `urlPiper = "http://localhost:5000"` to the piper HTTP container and that's probalby most of the stuff you need to to.

You can run the file with or without a argument. If you're not runnig it with a argument the script itself will tell you what it wants.

Run the script with
`python piper-tts.py "Hello world!"` 
to see what happens

**Comment by hairygael** [comment]
I ran the script in VS Code but got an error. It seems I am missing something.
`Traceback` (most recent call last):
  File "d:\Programs\piper\piper-tts.py", line 3, in <module>
    import requests
ModuleNotFoundError: No module named 'requests'``

**Comment by artibex** [comment]
I updated the client sided testing file to use available audio systems insted of a python package
